### PR TITLE
Static UI for find Purchase order screen(#1wn3fca)

### PR DESCRIPTION
**Pull request by Utkarshkaraiya**
localhost_8100_Purchaseorder
localhost_8100_Purchaseorder (1)

### CODE REVIEW

File: src/theme/variables.css [hunk 1]
```diff
@@ -240,6 +240,10 @@ http://ionicframework.com/docs/theming/ */
   grid-template-areas: "search"
                        "main";
 }
+.find > .filters{
+  display: none;
+
```
Comment (azkyakhan) [inline src/theme/variables.css]: extra space

File: src/views/PurchaseOrder.vue [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+    </ion-header>
+
+    <ion-content>
+      <div class="find">
+        <section class="search">
+          <ion-searchbar />
+        </section>
+     
+        <aside class="filters">
+
```
Comment (azkyakhan) [inline src/views/PurchaseOrder.vue]: extra space

File: src/views/PurchaseOrder.vue [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+              </ion-item>
+              <ion-item>
+                <ion-label>{{ $t("Facility") }}</ion-label>
+                <ion-select value="any">
+                  <ion-select-option value="any">California Warehouse</ion-select-option>
+                </ion-select>
+              </ion-item>
+           </ion-list>
+           <ion-list>
+               <ion-list-header>{{ $t("Date") }}</ion-list-header>
```
Comment (azkyakhan) [inline src/views/PurchaseOrder.vue]: improve indentation

File: src/views/PurchaseOrder.vue [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+  IonSearchbar,
+  IonSelect,
+  IonSelectOption,
+  IonThumbnail,
+  IonToggle,
+  IonTitle,
+  IonToolbar,
+} from "@ionic/vue";
+import {
+      calendarOutline,
```
Comment (azkyakhan) [inline src/views/PurchaseOrder.vue]: improve indentation

File: src/views/PurchaseOrder.vue [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+  grid-area: vtags;
+  justify-self: center;
+}
+
+.variant-metadata {
+  grid-area: vmetadata;
+}
+
+@media (min-width: 991px) {
+
```
Comment (azkyakhan) [inline src/views/PurchaseOrder.vue]: remove extra space and improve indentation

File: src/views/PurchaseOrder.vue [hunk 6]
```diff
@@ -0,0 +1,275 @@
+<template>
```
Comment (azkyakhan) [inline src/views/PurchaseOrder.vue:1]: Please go through the extra space and indentation of the page

**Review by azkyakhan — DISMISSED** [review]

File: src/theme/variables.css (hunk 1 above)
Comment (Utkarshkaraiya) [reply src/theme/variables.css]: Done maam.

File: src/views/PurchaseOrder.vue (hunk 2 above)
Comment (Utkarshkaraiya) [reply src/views/PurchaseOrder.vue]: done maam

File: src/views/PurchaseOrder.vue (hunk 3 above)
Comment (Utkarshkaraiya) [reply src/views/PurchaseOrder.vue]: Done maam

**Pull request merged**